bodhi/server/models.py

```diff
@@ -1154,6 +1154,36 @@ def get_pkg_committers_from_pagure(self):
         # contains FAS group names with commit access.
         return list(committers), list(groups)
 
+    def hascommitaccess(self, username: str, branchname: str) -> bool:
+        """
+        Check on Pagure if a user has commit access on the package/branch.
+
+        Raises:
+            RuntimeError: If Pagure did not give us a 200 code.
+        """
+        pagure_url = config.get('pagure_url')
+        # Pagure uses plural names for its namespaces such as "rpms" except for
+        # container. Flatpaks were moved from 'modules' to 'flatpaks' - hence
+        # a config setting.
+        if self.type == ContentType.container:
+            namespace = self.type.name
+        elif self.type == ContentType.flatpak:
+            namespace = config.get('pagure_flatpak_namespace')
+            # branchname override for all releases
+            branchname = config.get('pagure_flatpak_main_branch')
+        elif self.type == ContentType.module:
+            namespace = self.type.name + 's'
+            # There's no direct map between stream and branch names so override branchname
+            # for the moment
+            branchname = config.get('pagure_module_main_branch')
+        else:
+            namespace = self.type.name + 's'
```

Comment:
Food for thoughts, we could do something like: https://pagure.io/fedora-infra/toddlers/blob/main/f/toddlers.toml.example#_135 (ie: a dict that maps the ContentType.name to the dist-git names), this way if there is ever another namespace rename in the future, we can simply adjust the configuration file without having to touch the code.

We could have something like: ``namespace = namespace_dict[self.type.name]`` where ``namespace_dict`` is retrieved from the config.

Reply:
yup, good idea, I'll update the PR